marlowe-playground-client/static/css/css_var_colors.scss

```diff
@@ -2,7 +2,7 @@
   --white: white;
   --black: black;
   --primary-color: #832dc4;
-  --secondary-color: #1746a0;
+  --secondary-color: #1746a0; // TODO: replace with --blue
```

Comment:
why not do this work in this PR?

Reply:
I wanted to confirm the approach of having --blue-xxx in this PR before going ahead and modify all instances of this colors. Will do in a separate PR